src/lib/Handset/CRSFHandset.cpp

```diff
@@ -378,6 +381,24 @@ bool CRSFHandset::ProcessPacket()
         RcPacketToChannelsData();
         packetReceived = true;
     }
+    else if(packetType==CRSF_FRAMETYPE_MAVLINK_RAW) {
+        uint8_t size=SerialInBuffer[3];
+        mavlink_message_t msg;
+        mavlink_status_t status;
+        for(uint8_t i=0; i<size;i++)
+        {
+            if(mavlink_parse_char(MAVLINK_COMM_1,SerialInBuffer[4+i],&msg,&status))
```

Comment:
This use of `MAVLINK_COMM_1` will mean we need to bump `MAVLINK_COMM_NUM_BUFFERS` to 2. https://github.com/ExpressLRS/ExpressLRS/pull/2846